build.gradle

```diff
@@ -553,6 +553,7 @@ project(':core') {
     compile libs.scalaLogging
     compile libs.zkclient
     compile libs.zookeeper
+    compile libs.jacksonScalaModule
```

Comment:
Can we please avoid this dependency? I think it would be better to stick to plain Jackson.

Reply:
This dependency makes Jackson to decode scala objects, I am not fully sure but if we avoid this then we need to use the `ObjectNode` way and that will generate more line of code.

Reply:
I think we'd just have to avoid Scala library classes. Since we are mostly using maps, it be enough to do `asJava` on the Scala collection. But that needs to be verified.

Reply:
Thats a good point indeed, thanks a lot. I tested `asJava` on Scala Maps, Lists and it worked as expected. I'll remove this dependency.